packages/grid/x-data-grid/src/colDef/gridDateOperators.ts

```diff
@@ -47,73 +47,73 @@ function buildApplyFilterFn(
 export const getGridDateOperators = (showTime?: boolean): GridFilterOperator<any, Date, any>[] => [
   {
     value: 'is',
-    getApplyFilterFn: (filterItem) => {
+    getApplyFilterFn: v7((filterItem) => {
       return buildApplyFilterFn(filterItem, (value1, value2) => value1 === value2, showTime);
-    },
+    }),
     InputComponent: GridFilterInputDate,
     InputComponentProps: { type: showTime ? 'datetime-local' : 'date' },
   },
   {
     value: 'not',
-    getApplyFilterFn: (filterItem) => {
+    getApplyFilterFn: v7((filterItem) => {
```

Comment:
Could you update the performance docs with a section describing how to make custom filters faster?

Reply:
Should I add the content in the filters section or the performance section? I'll link to it in the other place.

Reply:
It's up to you, I don't really have preference here